### DISCUSSION TITLE
Install fails to add enumerator to Bluetooth devices

### DISCUSSION THREAD

**Discussion by StreethawkUK** [nefarius/DsHidMini#267] [Support]
Hi Everyone - seeking some installation help.

I've installed BthPS3 v2.6.167 (setup-v2.6.174) for .x64 using the all-in-one setup file. 
I'm trying to use it with DsHidMini v2.2.282.0 which I've installed using the detailed instructions and works flawlessly (less the BT support) - recognised by DSHMC and plays a random Epic game while plugged in using a DS3.

Problem I've got is there's no Nefarius device appeared under BT in Windows settings. 
![image](https://user-images.githubusercontent.com/120650133/207864699-1635bb61-c8ed-4c2a-a2f4-8c2697e1d8df.png)
I've been working through the step-by-step instructions and found mine does not appear the same and could use a pointer or two please anyone?...
![image](https://user-images.githubusercontent.com/120650133/207865670-c033494b-fcf2-488c-a948-216130572e12.png)

Couldn't see any similar threads but prev. exp. shows I'm rather pants at that ;)

**Comment by nefarius** [comment]
Is the setup itself finished successfully? Share the file `C:\Windows\INF\setupapi.dev.log` please.

**Reply by StreethawkUK** [reply]
I'm pretty sure the answer is Yes = although it's a long .log file every stage seems to end with SUCCESS
[setupapi.dev.log](https://github.com/ViGEm/DsHidMini/files/10240014/setupapi.dev.log)

**Reply by nefarius** [reply]
Hm, looking at line `19203` it appears that the installation was aborted somehow and didn't even write a finished log, can't really tell you what happened there 🤔

**Reply by StreethawkUK** [reply]
That is peculiar - certainly there was no user interference!

I can run it again but nothing else has changed so i wouldn't expect a different outcome. 
it's a clean Win10 install on a new machine. Any other advice short of getting some longer USB cables (which is all i have ATM)?

**Reply by nefarius** [reply]
[Run this tool](https://github.com/nefarius/Legacinator/releases/tag/v1.1.42) then reinstall and make extra sure the setup wizard finishes without any errors.

**Reply by StreethawkUK** [reply]
When I run Legacinator, BThPS3 drivers are found but not removed when i select the tile - it greys out briefly, the waity wheel turns but the tile just remains there. 
Closing it and running again to check if the drivers remain and nothing has changed...

**Reply by nefarius** [reply]
Oh dear, well it is a beta release after all, will try to address it when I find the time.

**Reply by StreethawkUK** [reply]
Coolio - good luck. Let me know when you've had some time to fiddle with it and I'll test it our for you :)

**Reply by nefarius** [reply]
Did it generate a `Legacinator.log` file? If so could you please attach it here, thx.
